sympy/printing/fortran.py

```diff
@@ -387,8 +387,7 @@ def _print_AugmentedAssignment(self, expr):
         lhs_code = self._print(expr.lhs)
         rhs_code = self._print(expr.rhs)
         return self._get_statement("{0} = {0} {1} {2}".format(
-            *map(lambda arg: self._print(arg),
-                 [lhs_code, expr.binop, rhs_code])))
+            *(self._print(arg) for arg in [lhs_code, expr.binop, rhs_code])))
```

Comment:
I don't think that this 3 element loop is worth it either. Better to make is explicit:
```
"{0} = {0} {1} {2}".format(self._print(lhs_code), self._print(expr.binop), self._print(rhs_code))
```
Otherwise I would use `map` but the `lambda` here is unnecessary:
```
"{0} = {0} {1} {2}".format(map(self._print, [lhs_code, expr.binop, rhs_code]))
```